try:
    print("hello")
except:
    print("exception happend")
